package.json

```diff
@@ -89,9 +88,10 @@
     "jest": "^25.0.0",
     "jest-cli": "^25.0.0",
     "markdown-loader-jest": "^0.1.1",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
-    "react-test-renderer": "^16.14.0",
+    "prop-types": "^15.8.1",
```

Comment:
We already have `prop-types` package in dependencies, I don't think we need it as a dev dependency too

Reply:
I agree, remove